core/src/main/java/org/arquillian/smart/testing/configuration/ObjectMapper.java

```diff
@@ -223,6 +227,23 @@ private static Object handleList(Method method, Object mappedValue) {
                 convertedList.add((T) convertToType(parameterType, v));
             }
             return convertedList;
+        } else if (Map.class.isAssignableFrom(aClass) && StrategyConfiguration.class.isAssignableFrom(parameterType)) {
+            final Map<String, Object> objectMap = (Map<String, Object>) mappedValue;
+            List<T> convertedList = new ArrayList<>(objectMap.size());
+            objectMap.forEach((k, v) -> {
+                final JavaSPILoader javaSPILoader = new JavaSPILoader();
+                final Optional<StrategyConfiguration> strategyConfiguration =
+                    javaSPILoader.onlyOne(StrategyConfiguration.class, config -> config.name().equals(k));
```

Comment:
I'm a bit confused here - this seems to be a duplication of the logic in `Configuration.strategyConfigurations`.
Maybe appropriate naming of the variables & refactoring would help...